src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/DateTimeValueResolverTest.php

```diff
@@ -132,7 +132,6 @@ public function testNow(string $timezone)
 
         /** @var \Generator $results */
         $results = $resolver->resolve($request, $argument);
-        $results = iterator_to_array($results);
```

Comment:
So, `$results` already is an array? In that case, the `@var` inline doc block is also wrong, isn't it?

Reply:
indeed, fixed